tts-deploy-on-embedded.ipynb

```diff
@@ -0,0 +1,265 @@
+   "id": "99c24ffc",
+   "metadata": {},
+   "source": [
+    "### Download models\n",
+    "We will download pretrained [Fastpitch](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechsynthesis_en_us_fastpitch_ipa) and [HifiGan](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechsynthesis_en_us_hifigan_ipa) models from ngc. You can replace these models with the paths of your custom model, incase of custom models."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "be2499b8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!ngc registry model download-version \"nvidia/tao/speechsynthesis_en_us_fastpitch_ipa:deployable_v1.0\"\n",
+    "!ngc registry model download-version \"nvidia/tao/speechsynthesis_en_us_hifigan_ipa:deployable_v1.0\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "19d3303d",
+   "metadata": {},
+   "source": [
+    "### Set configs and params.\n",
+    "Set following config parameters:  \n",
+    "`synthesizer`: Full path for synthesizer.riva file from [ngc](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechsynthesis_en_us_fastpitch_ipa)  \n",
+    "`vocoder`: Full path for vocoder.riva file file from [ngc](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechsynthesis_en_us_hifigan_ipa)  \n",
+    "`out_dir`: Directory to put TTS.rmir file  \n",
+    "`voice`: Set the voice name of the model.  \n",
+    "`key`: This is the encryption key used in nemo2riva. Same key will be used to deploy the RMIR generated in this tutorial.  \n",
+    "`use_ipa`: Set to \"y\" or \"Y\" if the model uses IPA phones, \"no\" if the model uses arpabet.  \n",
+    "`lang`: Model language.  \n",
+    "`sample_rate`: Sample rate of generated audios.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "579b6420",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "synthesizer = pathlib.Path.cwd() / \"speechsynthesis_en_us_fastpitch_ipa_vdeployable_v1.0/FastPitch_44k_EnglishUS_IPA.riva\" ##Synthesizer .riva location\n",
+    "vocoder = pathlib.Path.cwd() / \"speechsynthesis_en_us_hifigan_ipa_vdeployable_v1.0/HifiGAN_44k_EnglishUS_IPA.riva\" ##vocoder .riva location\n",
+    "out_dir = pathlib.Path(\"out/\") ##Output directory to store generated RMIR.\n",
+    "voice = \"test\" ##Voice name\n",
+    "key = \"tlt_encode\" ##Encryption key used during nemo2riva\n",
+    "use_ipa = \"no\" ##\"y\" or \"Y\" if the model uses ipa, no otherwise.\n",
+    "lang = \"en-US\" ##Language\n",
+    "sample_rate = 44100 ##Sample rate of the audios\n",
+    "\n",
+    "\n",
+    "## Riva NGC, servicemaker image config.\n",
+    "riva_ngc_org = \"nvidia\"\n",
+    "riva_ngc_team = \"riva\"\n",
+    "NGC_TARGET = f\"{riva_ngc_org}/{riva_ngc_team}\"\n",
+    "riva_ngc_image_version = \"2.8.0\"\n",
+    "riva_init_image = f\"nvcr.io/{NGC_TARGET}/riva-speech:{riva_ngc_image_version}-servicemaker-l4t-aarch64\""
```

Comment:
You should point out what this is and what it means. Are you sure that you want to download the `aarch64` version? This depends on the machine that this notebook is being run on.

Reply:
Since the tutorial is about deploying on embedded. I can add a flag in config and use the image accordingly.